dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -397,12 +400,14 @@ public void visualizeBlockTransferInput(Block bb, Analysis<A, S, T> analysis) {
 
     @Override
     public void visualizeBlockNode(Node t, /*@Nullable*/ Analysis<A, S, T> analysis) {
-        A value = analysis.getValue(t);
-        String valueInfo = "";
-        if (value != null) {
-            valueInfo = "    > " + prepareString(value.toString());
+        this.sbBlock.append(prepareString(t.toString()) + "   [ " + prepareNodeType(t) + " ]");
+        if (analysis != null) {
+            A value = null;
+            value = analysis.getValue(t);
```

Comment:
Combine this line with the previous one.  If you were going to put them on separate lines, there would be no point in initializing `value` in its declaration, because it will be immediately overwritten.  It's even clearer to assign `value` to its real value at its declaration.